.github/workflows/deploy.yml

```
@@ -19,9 +19,21 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: '3.5'
+    - name: Install GCC (Linux)
+      run: |
+        sudo apt update
+        sudo apt-get install -y make ${{ matrix.gcc }} $(echo ${{ matrix.gcc }} | sed -e 's/gcc/g\+\+/')
+        sudo apt-get clean
     - name: Install dependencies
       run: |
         make setup-dev PYTHON=python
+    - name: Lint with flake8
+      run: |
+        make lint PYTHON=python
+        admin/release_checklist --no-verify-tags --no-verify-docs-next-version 4.2
```

For the release, all items in the checklist should be enabled:

```suggestion
        admin/release_checklist 4.2
```

This will require a corresponding update in the checklist itself.

---

I copied this from Travis but you're fight.

---

I've added the checklist also to the test.yml (with the two options) and removed the options from the deploy.yml. checklist is also updated to scan both YML files.